alibi_detect/utils/pytorch/kernels.py

```diff
@@ -20,21 +48,122 @@ def sigma_median(x: torch.Tensor, y: torch.Tensor, dist: torch.Tensor) -> torch.
 
     Returns
     -------
-    The computed bandwidth, `sigma`.
+    The logrithm of the computed bandwidth, `log-sigma`.
     """
     n = min(x.shape[0], y.shape[0])
     n = n if (x[:n] == y[:n]).all() and x.shape == y.shape else 0
     n_median = n + (np.prod(dist.shape) - n) // 2 - 1
     sigma = (.5 * dist.flatten().sort().values[n_median].unsqueeze(dim=-1)) ** .5
-    return sigma
+    return sigma.log()
 
 
-class GaussianRBF(nn.Module):
+class KernelParameter(object):
+    """
+    Parameter class for kernels.
+    """
```

Comment:
General remark - missing constructor docstring for parameters here and in other classes.

@ascillitoe did we have a convention documenting class constructors either at the class level or at the `__init__` level? Does it matter for docs rendering?

Reply:
[PEP 257](https://peps.python.org/pep-0257/) says:

> The class constructor should be documented in the docstring for its __init__ method.

But then the [NumPy style guide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) says:


> The __init__ method may be documented in either the class level
> docstring, or as a docstring on the __init__ method itself.
> 
>  Either form is acceptable, but the two should not be mixed. Choose one
>  convention to document the __init__ method and be consistent with it.

We usually put the docstring in `__init__` so I guess you could say that is our convention...

Reply:
P.s. I shall add a note to `CONTRIBUTING.md`